### PR TITLE
chore: fix re-introduced ref sibling validation error

### PR DESCRIPTION
**Pull request by kleyow**
I had re-introduced this ref/sibling validation error when coverting over to openapi-cli and re-spliting the fspiop apis.
This is blocking my work. I also need to do this for fspiop v1.0 but thats lower priority.

```
This is required to pass the validation check for ‘openapi-backend’.

All of the removed examples have been moved to the base schema. However, the removed descriptions have not been handled.
```
https://github.com/mojaloop/api-snippets/commit/316caf9cbcb5e7eec5bd782e8ca5ee3fba615a22#diff-8e437e159322e58959371fbff3e4430b7565eeeace02bebd723400e6ebb40609

Future note. OpenApi 3.1.0 should support $ref description and example siblings. Maybe when support for that is widespread I can re-add the descriptions back...

### CODE REVIEW

File: fspiop/v1_0/openapi3/components/schemas/Amount.yaml [hunk 1]
```diff
@@ -8,3 +8,4 @@ description: >-
   minor currency unit. It also only allows four digits in the minor currency
   unit; a negative value is not allowed. Using more than 18 digits in the major
   currency unit is not allowed.
+example: 123.45
```
Comment (eoln) [inline fspiop/v1_0/openapi3/components/schemas/Amount.yaml:11]: there is little inconsistency over the examples: sometimes string field example value is in quotes, sometimes not. I suggest using the quotes for all strings to distinguish values from numbers

**Review by eoln — CHANGES_REQUESTED** [review]

File: fspiop/v1_0/openapi3/components/schemas/Amount.yaml (hunk 1 above)
Comment (kleyow) [reply fspiop/v1_0/openapi3/components/schemas/Amount.yaml]: thanks for pointing this out. done

File: docs/fspiop-rest-v1.1-openapi3-snippets.yaml [hunk 2]
```diff
@@ -2792,6 +2805,7 @@ components:
       minLength: 1
       maxLength: 128
       description: Memo assigned to transaction.
+      example: Note sent to Payee.
```
Comment (eoln) [inline docs/fspiop-rest-v1.1-openapi3-snippets.yaml:2808]:
```suggestion
      example: 'Note sent to Payee.'
```

File: docs/fspiop-rest-v1.1-openapi3-snippets.yaml [hunk 3]
```diff
@@ -3007,6 +3024,7 @@ components:
         - OTP - One-time password generated by the Payer FSP.
         - QRCODE - QR code used as One Time Password.
         - U2F - U2F is a new addition isolated to Thirdparty stream.
+      example: OTP
```
Comment (eoln) [inline docs/fspiop-rest-v1.1-openapi3-snippets.yaml:3027]:
```suggestion
      example: 'OTP'
```

**Review by eoln — APPROVED** [review]

**Comment by eoln** [comment]
looks like in generated file the quotes disappeared...

**Comment by kleyow** [comment]
Ya...openapi-cli takes a bit of decision making for itself.

**Pull request merged**